AirCasting/SessionViews/SessionCartViewModel/MeasurementsViewModel.swift

```diff
@@ -0,0 +1,60 @@
+// Created by Lunar on 11/09/2021.
+//
+
+import Foundation
+import CoreLocation
+import CoreData
+
+class ViewModel: NSObject, ObservableObject {
```

Comment:
The name should be probably something more than a `ViewModel`. Maybe `MeasurementsViewModel` or even more verbose `CardMeasurementsViewModel`?

Also is `NSObject` needed?

Reply:
Right, thanks, dunno why I left it named ViewModel :D